### PR TITLE
virsh_cpu_models: restart libvirtd for remote test

### PR DESCRIPTION
**Pull request by cliping**
Restart remotely libvirtd to avoid test failed.

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type i440fx virsh.cpu_models.positive_test.remote_host
JOB ID     : 3eb6bb2d1344bb8ac1e0c3890482cc381aec8dc3
JOB LOG    : /root/avocado/job-results/job-2018-01-04T03.19-3eb6bb2/job.log
 (1/2) type_specific.io-github-autotest-libvirt.virsh.cpu_models.positive_test.remote_host.auto_get_arch: PASS (22.43 s)
 (2/2) type_specific.io-github-autotest-libvirt.virsh.cpu_models.positive_test.remote_host.specific_arch: PASS (21.79 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 47.33 s

File: libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py [hunk 1]
```diff
@@ -16,10 +21,41 @@ def run(test, params, env):
...
+    restart_libvirtd_remotely = "yes" == params.get(
+        "restart_libvirtd_remotely", "no")
+
+    remote_ip = params.get("remote_ip", "REMOTE.EXAMPLE.COM")
+    remote_pwd = params.get("remote_pwd", None)
+
+    local_ip = params.get("local_ip", "LOCAL.EXAMPLE.COM")
+    local_pwd = params.get("local_pwd", None)
+
+    logging.debug(target_uri.count('EXAMPLE.COM'),
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py]: if you want to debug output, better add keywords here ,e.g "xx %d, target_uri.count"

File: libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py [hunk 2]
```diff
@@ -16,10 +21,41 @@ def run(test, params, env):
...
+                  remote_ip.count('EXAMPLE.COM'),
+                  local_ip.count('EXAMPLE.COM'))
+    if (target_uri.count('EXAMPLE.COM') or remote_ip.count('EXAMPLE.COM') or
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py]: suggest use 'EXAMPLE.COM' in (target_uri, remote_ip,xx) to simplify this statement

File: libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py [hunk 3]
```diff
@@ -16,10 +21,41 @@ def run(test, params, env):
...
+            local_ip.count('EXAMPLE.COM')):
+        raise error.TestNAError("Please replace '%s/%s/%s' with valid uri or "
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py]: remove this autotest dependence

File: libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py [hunk 4]
```diff
@@ -16,10 +21,41 @@ def run(test, params, env):
...
+        try:
+            session = remote.remote_login("ssh", remote_ip, "22", "root",
+                                          remote_pwd, "#")
+            session.cmd_output('LANG=C')
+            ssh_key.setup_remote_ssh_key(remote_ip, "root", remote_pwd,
+                                         local_ip, "root", local_pwd)
+
+            cmd = "service libvirtd restart"
+            status, output = session.cmd_status_output(cmd, internal_timeout=5)
+            logging.debug("cmd: %s" % cmd)
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py]: maybe merge this debug into the following statement

File: libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py [hunk 5]
```diff
@@ -16,10 +21,41 @@ def run(test, params, env):
...
+            logging.debug("status: %s, output: %s" % (status, output))
+        except process.CmdError, info:
+            raise exceptions.TestError("Error to run on remote: %s" % info)
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py]: remove autotest dependency here

**Comment by cliping** [comment]
@chunfuwen , done.

**Pull request merged**